### PR TITLE
[tflite2circle] support SVDF for circle conversion

### PR DESCRIPTION
**Pull request by BalyshevArtem**
This pr enables SVDF Op for circle conversion

issue #8203 

ONE-DCO-1.0-Signed-off-by: Artem Balyshev a.balyshev@partner.samsung.com

### CODE REVIEW

File: compiler/tflite2circle/src/BuildBuiltinOptions/SVDFOptions.h [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __BBO_SVDF_OPTIONS_H
+#define __BBO_SVDF_OPTIONS_H
```
Comment (m-bronnikov) [inline compiler/tflite2circle/src/BuildBuiltinOptions/SVDFOptions.h:18]: Could you please use `__BBO_SVDF_OPTIONS_H__` in accordance with other options headers?

Comment (BalyshevArtem) [reply compiler/tflite2circle/src/BuildBuiltinOptions/SVDFOptions.h]: Thank you, done

**Comment by m-bronnikov** [comment]
LGTM, Thank you!

----------------

P.S. Please don't rebase from next time =)

**Review by binarman — APPROVED** [review]
LGTM

**Comment by seanshpark** [comment]
Code looks OK but I'd like to see full working draft that pass the test.
Please refer https://github.com/Samsung/ONE/issues/8203#issuecomment-999933855

**Comment by seanshpark** [comment]
@binarman , thanks for calling me :)

**Comment by m-bronnikov** [comment]
> Code looks OK but I'd like to see full working draft that pass the test. Please refer [#8203 (comment)](https://github.com/Samsung/ONE/issues/8203#issuecomment-999933855)

@seanshpark Draft for SVDF op looks ready (#8231). With this draft tflite models with `SVDF` could be successfully converted to circle, imported to luci and executed by luci-interpeter. Can we approve this chages?

+cc @BalyshevArtem

**Comment by seanshpark** [comment]
I think draft is ready. If you want to verify value testing, please add to `luci-value-test/` list.

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Pull request merged**